### DISCUSSION TITLE
How to use scattered data in training process with DeepXDE?data

### DISCUSSION THREAD

**Discussion by rahaarbasi** [lululxvi/deepxde#1362] [Q&A]
Dear Professor Lulu @lululxvi,

I am trying to use DeepXDE to solve a PDE with initial and boundary conditions and collocation points. I also have scattered data that I want to use in the training process. However, I am having trouble incorporating the scattered data into the training process.

I tried using the `train_x` and `train_y` arguments when defining my training data with the `dde.data.PDE` class, but it didn't work. Could you please provide some guidance on how to use scattered data in the training process with DeepXDE?

Thank you for your help!

**Comment by lululxvi** [comment]
See inverse problem examples.
